libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -4,6 +4,7 @@ include AP_Common/Location.h
 
 userdata Location field lat int32_t read write -900000000 900000000
 userdata Location field lng int32_t read write -1800000000 1800000000
+userdata Location field alt int32_t read write -INT32_MAX INT32_MAX
```

Comment:
The max should be LOCATION_ALT_MAX_M, the minimum should probably be -((1 << 24) - 1) since we periodically need to be able to serialize this to 24 bit for missions.

Reply:
How about this?  I think this will match [our existing check in AP_Mission](https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Mission/AP_Mission.cpp#L1003)

> userdata Location field alt int32_t read write (-LOCATION_ALT_MAX_M*100+1) (LOCATION_ALT_MAX_M*100-1)